config/config.go

```diff
@@ -0,0 +1,16 @@
+package config
```

Comment:
This is just to support other things, just boilerplace that will get removed with the config loading